rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -2527,6 +2527,44 @@ private Map<String, Object> createPaginationMetadata(Pageable pageable, Map<Stri
         return responseBody;
     }
 
+    @PreAuthorize("hasAuthority('WRITE')")
+    @Operation(
+            summary = "delete orphan obligations",
+            description = "Pass an array of orphan obligation ids in request body.",
```

Comment:
Pls change this to obligation titles instead of ids.

Reply:
done.